.pylintrc

```diff
@@ -23,4 +23,8 @@ disable=bad-continuation,
   too-few-public-methods,
   too-many-ancestors,
   too-many-instance-attributes,
-  duplicate-code
+  duplicate-code,
+  too-many-arguments,
+  too-many-locals,
+  too-many-statements,
+  broad-except
```

Comment:
IMHO these checks are useful to have on by default as they can flag less-than-ideal code. I'd prefer to have them enabled globally and turned off on a case-by-case basis (as IIRC you had it in an earlier iteration?). 

Note that duplicate-code complaints about the `import` section at the top of various files, which are typically noise, can be generally avoided with:

```ini
[SIMILARITIES]

ignore-imports=yes
```

which I'm in favor of adding here.